File: CodeChef/Beginner/06FirstAndLastDigit/firstandlast.java
import java.util.Scanner;

class firstandlast {

    public static int Sum(int x) {
        int sum, a = 0;
        sum = (x % 10);
        while (x != 0) {
            a = (x % 10);
            x /= 10;
        }
        sum += a;
        return sum;
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int n, x;
        n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            System.out.println(Sum(sc.nextInt()));
        }
        sc.close();

    }

}

File: CodeChef/Beginner/06FirstAndLastDigit/firstandlast.cpp
#include <iostream>
using namespace std;
class firstandlast {
public:
  int indexsum(int x) {
    int sum;
    int a;
    sum = (x % 10);
    while (x != 0) {
      a = x % 10;
      x /= 10;
    }
    sum += a;
    return sum;
  }
};

int main() {
  int n, x;
  cin >> n;

  firstandlast flobj = firstandlast();
  for (int i = 0; i < n; i++) {
    cin >> x;
    cout << flobj.indexsum(x) << endl;
  }
  return 0;
}